sources/niLang/src/Expression.cpp

```diff
@@ -5949,14 +5949,14 @@ tBool Evaluator::TokenizeExpr(const achar* aaszExpr, tMathExprTokenVec& avOut)
       }
 
       token.Type = eMathExprTokenType_Float;
-      if (varNum.GetType() == eType_U32) {
-        token.fFloat = varNum.mU32;
+      if (varNum.GetType() == eType_U64) {
```

Comment:
Looks like a side effect of the changes in ReadNumberEx, looks correct but I wish I didnt write the code this way originally ;)

Reply:
I think we need to keep both and this should become a `switch/case` on `varNum.GetType()`

Reply:
We also have to audit if they are any other "if eType_U/I/F32" in here.